circuits/cpp/src/aztec3/constants.hpp

```diff
@@ -59,9 +60,11 @@ constexpr size_t MAX_PUBLIC_DATA_UPDATE_REQUESTS_PER_TX = 4;
 constexpr size_t MAX_PUBLIC_DATA_READS_PER_TX = 4;
 constexpr size_t MAX_NEW_CONTRACTS_PER_TX = 1;
 constexpr size_t MAX_OPTIONALLY_REVEALED_DATA_LENGTH_PER_TX = 4;
+constexpr size_t MAX_READ_REQUESTS_PER_TX = 4;
```

Comment:
Should this be: `MAX_PRIVATE_CALL_STACK_LENGTH_PER_CALL * MAX_READ_REQUESTS_PER_CALL `? Maybe we can add a TODO to increase it in near future.